utils/kata-manager.sh

```diff
@@ -260,11 +260,13 @@ pre_checks()
 	command -v "${kata_shim_v2}" &>/dev/null \
 		&& die "Please remove existing $kata_project installation"
 
+	[only_kata = "false" ] && return 0
+
 	local ret
 
 	{ containerd_installed; ret=$?; } || true
 
-	[ "$ret" -eq 0 ] && [only_kata = "false" ] && die "$containerd_project already installed"
+	[ "$ret" -eq 0 ] && die "$containerd_project already installed"
```

Comment:
@gabevenberg Can you squash this commit with the previous one?

Reply:
done.